scrapers/covid-19/getVaccine.js

```diff
@@ -173,6 +183,65 @@ const generateSpecificCountryVaccineData = (timelineData) => {
 	return { country: '', countryInfo: null, timeline: updatedTimeline };
 };
 
+/**
+ * Formats raw timeline data
+ * @param {array} timelineData An array of raw timeline data
+ * @returns {object}
+ */
+
+const generateSpecificStateVaccineData = (timelineData) => {
```

Comment:
this method copies a ton of code from the above method, is there any way you can merge them into one generic method to reuse the code?

Reply:
Sure.